USFMToolsSharp/Models/Markers/FIGMarker.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace USFMToolsSharp.Models.Markers
+{
+
+    /// <summary>
+    /// Wordlist / Glossary / Dictionary Entry Marker
+    /// </summary>
+    public class FIGMarker : Marker
+    {
+        public string Caption;
+        public string Description;
+        public string Width;
+        public string Location;
+        public string Copyright;
+        public string Reference;
+        public string FilePath;
+
+        public override string Identifier => "fig";
+
+        public override string PreProcess(string input)
```

Comment:
I would like this to support USFM 3 as well. Because we have no idea if people are going to be using it nor not.

Reply:
Ah...didn't notice that it was version 1/2 only.  The document that I had as an example obviously was the older format.  I'll update this next week.

Reply:
@rbnswartz my colleague here worked on the changes this morning.  \fig now should be USFM version 1-3 ready.  He even threw in a few unit tests.

One thing we noticed is that the unit test framework is old on the solution:

```
    <PackageReference Include="MSTest.TestAdapter" Version="1.4.0" />
    <PackageReference Include="MSTest.TestFramework" Version="1.4.0" />
```

Both of these can be updated to version 2.2.4 which are "MSTest V2, the evolution of Microsoft's Test Framework".  On my system, I needed to update these to the new version to run the tests.  My colleague did not.  Is there any reason why these should not be updated?

Reply:
@rbnswartz regardless of the above, all the changes you requested should be resolved now.